### PR TITLE
Feature/l3 100

### PR DESCRIPTION
**Pull request by Ellenn-A**
Hi Guys, 

Could you pls have a look on the changes I have made? 
I have separated the seed data into separate files that tests use. 

Also sth odd is going on with the linting on the files I have added pls let me know how to fix that. 

For chain.test.ts there was the seed file being run before the test, but I have removed it as we don't need any seed data for that test. I replaced it with cleanup() but I'm not sure if we need that either. 

I have left the index file (original seed file) in, but it's not used anywhere anymore. Would you want me to remove it? 

also do I want to keep version as 1.2.43?

### CODE REVIEW

File: docker-compose-3-persona.yml [hunk 1]
```diff
@@ -77,6 +77,7 @@ services:
       --alice
       --unsafe-ws-external
       --unsafe-rpc-external
+      --ws-max-connections 512
```
Comment (dblane-digicatapult) [inline docker-compose-3-persona.yml:80]: The additional `ws-max-connections` argument is not needed in `docker-compose-3-persona.yml` as we do not run automated tests against this file.

File: test/seeds/index.ts [hunk 2]
```diff
@@ -1,493 +1,493 @@
...
+
+// const db = new Database().db()
+
+// export const cleanup = async () => {
+//   await db.attachment().del()
+//   await db.demand().del()
+//   await db.transaction().del()
+//   await db.match2().del()
+//   await db.demand_comment().del()
+// }
```
Comment (dblane-digicatapult) [inline test/seeds/index.ts:12]: Delete file.

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]
Looks good.  Try running the linter with the fix argument to fix the linting.
Delete the old seed index file.

**Review by mattdean-digicatapult — COMMENTED** [review]
looks like a big improvement to me. Happy once @dblane-digicatapult is happy

**Review by dblane-digicatapult — APPROVED** [review]

**Pull request merged**